src/Menu/SubheaderMenuItem.jsx

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-var SubheaderMenuItem = React.createClass({
-    
-    propTypes: {
-        index: React.PropTypes.number.isRequired,
-        text: React.PropTypes.string.isRequired
-    },
-    
-    render: function() {
-        return (
-            <div key={this.props.index} className="chamel-subheader">{this.props.text}</div>
-        );
-    }
-    
-});
+const SubheaderMenuItem = (props) => {
+  return (
+    <div key={props.index} className="chamel-subheader">{props.text}</div>
```

Comment:
I know it was this way before, but setting className to 'chamel-subheader' will not work with css modules. You need to import the css and apply the classname like the rest of the components.